### PR TITLE
Updated/verified file names, literal values, emphasis, lists, and ref…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…erences to RH guides

### CODE REVIEW

File: asciidoc-markup-sample-doc/content/asciidoc-markup-samples.adoc [hunk 1]
```diff
@@ -57,13 +57,13 @@ a|use [command]*yum to install new packages.
 a|$$_emphasis_$$
 a|Use _this_ approach.
 
-|Filenames or directory paths
+|File names or directory paths
```
Comment (samccann) [inline asciidoc-markup-sample-doc/content/asciidoc-markup-samples.adoc]: We need semantic tags for file names and paths, per the FATE/emender test team. See the later column in the spreadsheet for details - [filename]`file` etc.

Comment (bergerhoffer) [inline asciidoc-markup-sample-doc/content/asciidoc-markup-samples.adoc]: @samccann Is this really a priority right now? It seems like we should be focusing on the presentation, and the syntax to achive that, for now, and perhaps worry about the emender testing semantics later. Maybe some teams are ready for these semantic tagging, but I don't think these are a requirement right now as far as I understand.

Comment (samccann) [inline asciidoc-markup-sample-doc/content/asciidoc-markup-samples.adoc]: as I recall, we decided to incorporate emender requirements at the start. That said, I don't know what it takes to get asciidoc tools etc to use these new semantic requirements. @rkratky - can you help here? I don't want to slow things down, but I also know RHEL and I think maybe Ceph docs depend on these tags in docbook today, so we'd be impacting those teams if we don't support emender testing...

Comment (rkratky) [inline asciidoc-markup-sample-doc/content/asciidoc-markup-samples.adoc]: @bergerhoffer, @samccann, you're both right :-)

The purpose of this initiative is to fix the _presentation_ style, and the semantic syntax is not crucial for that. However, as we will be providing *mark-up* guidance, we agreed to include the semantics, too. The semantic marking is a purely back-end thing, and it doesn't have any bearing over the actual content, so I think we can safely say it's our turf, too.

The sooner writers start using the semantic mark-up, the better. Even if the automated testing tools aren't ready to make use it yet, it doesn't matter. When they are, we'll have a head start.

**Pull request merged**